Client/Ecosia/UI/NTP/Cells/NTPLibraryCell.swift

```diff
@@ -46,17 +59,20 @@ class NTPLibraryCell: UICollectionViewCell, NotificationThemeable, ReusableCell
         heightConstraint.isActive = true
         self.heightConstraint = heightConstraint
 
-
         // Ecosia: Show history instead of synced tabs
-        [bookmarks, history, readingList, downloads].forEach { item in
+        Item.allCases.forEach { item in
             let view = NTPLibraryShortcutView()
             view.button.setImage(item.image, for: .normal)
-            view.button.tag = item.tag
+            view.button.tag = item.rawValue
```

Comment:
I believe that `tag` is more concise and self-explanatory at this precise line and avoid the double of the `Item` enum type.
What if we'd add the following underneath the `var image: UIImage?` ? 👇 
```
        var tag: Int {
            rawValue
        }
``` 
So that this line would become even more explicit and emphasize the outcome of your refactoring even more, leaving other parts of the codebase like this one, untouched.
```suggestion
            view.button.tag = item.tag
```

Reply:
I believe that if we still use `rawValue` inside `tag` it would still require `Int` as the `enum` type. Even so, I understand your suggestion and, thinking that this is set only to identify which button was tapped during the action, I don't think the word `tag` gives much more explanation.

Even so, your suggestion gave me an improvement idea, I will implement it and ask for your review again so you can give feedback if you agree 🙂